File: C program/even_or_odd.c
#include<stdio.h>
int main()
{
	int num,r;
	
	printf("Enter the number:\n");
	scanf("%d",&num);
	
	r=num%2;
	
	if(r==0)
	printf("%d is EVEN",num);
	if(r!=0)
	printf("%d is ODD",num);
}


/*
----------output----------
Enter the number:
8
8 is EVEN
*/

File: Java Programming/even_or_odd.java
import java.util.Scanner;
public class even_or_odd
{ 
    public static void main(String[] arg)
    {
        Scanner num = new Scanner(System.in);

        System.out.print("Enter any integer value: ");
        int value = num.nextInt();
        System.out.println("Entered value" + " = " + value);

        if(value%2==0)
        {
            System.out.print("Number is " + value + " and" + " is even.");
        }
        else
        {
            System.out.print("Number is " + value + " and" + " is odd.");
        }
        num.close();
    }   
}





/* 
--------------------output--------------------
Enter any integer value: 344
Entered value = 344
Number is 344 and is even.
*/